credentialdigger/client_postgres.py

```diff
@@ -373,6 +457,37 @@ def get_discovery_group(self, repo_url, state=None):
             query='SELECT file_name, snippet, count(id), state FROM \
             discoveries WHERE repo_url=%s GROUP BY file_name, snippet, state')
 
+    def get_embedding(self, discovery_id=None, snippet=None):
+        """ Retrieve a discovery embedding.
+
+        Parameters
+        ----------
+        discovery_id: int
+            The id of the discovery whose embedding is being retrieved
+        snippet: str
+            The snippet whose embedding is being retrieved. Only used if
+            discovery_id is not provided
+
+        Returns
+        -------
+        list
+            The embedding for the provided
```

Comment:
a list of embeddings (floats) ?

Reply:
yes I'm not sure how to specify the list type in the docstring